### ISSUE TITLE
NVDA stops reading elements like the start menu, text in notepad, etc at random times.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2403]
**Reported by santiagoblue7 on 2012-06-01 22:17**
Hello.
I'm currently using NVDA snapshot main 5207 with my iMac running Windows 8 release preview and I'm experiencing an issue with it. The issue is with NVDA not reading elements for example, it won't read the start menu, a page in firefox, text on notepad, etc. This happens at random times, so there's no given time when it will happen. Running as administrator won't fix the issue either. Restarting NVDA is the only thing that will temporarily fix this.

**Comment by ehollig** [comment]
@Brian1Gaff, have you experienced this lately? Considering I have not this issue and we do not have a way to contact the original author, I am leaning towards a close.

**Comment by Adriani90** [comment]
@Brian1Gaff, I cannot reproduce this in NVDA 2018.4 Beta 1 in Windows 10 1809 Update. Also working since two years with NVDA 2017.3 on Windows 8.1 and cannot reproduce this either.
Last year I had this problem in Windows 10 1703 update with NVDA 2017.2 and newer. But this problem seems to be solved after Windows 10 1803 update. @josephsl we have closed several issues related to this problem. I suggest closing this one as well unless someone can still reproduce it.

**Comment by ehollig** [comment]
As we have not experienced this issue in quite a bit of time, I am going to close this issue. Please reopen a new issue if something like this occurs in the future.
